scripts/config-dark.json

```diff
@@ -0,0 +1,37 @@
+{
+  "include": ["./src/asset/*.json", "./src/size/*.json", "./src/*.json"],
+  "source": ["./src/color-dark/*.json"],
+  "platforms": {
+    "SCSSDarkVariables": {
+      "transformGroup": "scss",
```

Comment:
We will need a transform group for standard CSS as well.

Reply:
What exactly do you mean by this? Do you want me to change this to `transformGroup: css`? I copied this code over exactly from the original `config.json` and in my latest patch, added some more `platform` groups that include a transform group for css. Let me know if that was what you were looking for or what I may be missing.

Reply:
There should be a CSS output also. More and more teams are moving to using the raw CSS from the CDN versus having to process Sass locally.